README.md

```diff
@@ -26,37 +26,45 @@ Future starts can be run as
 vagrant up
 ```
 
-To access the web interface outside of the vagrant container, you'll need to add certificates for nginx-proxy [as noted in the warehouse networking setup](https://github.com/greenriver/hmis-warehouse/blob/production/docs/developer-networking.md#certificate) and then restart vagrant. If you allowed vagrant to clone
-and create a default configuration, the certificates will be created for you,
-but you will still need to install them on your local machine.
+If initialization fails with errors related to pulling images, you may need to fix the DNS configuration. See the next section.
 
 
-If you want both CAS and the warehouse to share a database server, you may want to add a `docker-compose.override.yml` to one of the warehouse installation that puts the db container on a different port.
-```yaml
-version: '3.8'
+Once Vagrant is up, `ssh` in and run the setup scripts:
+```sh
+vagrant ssh
+cd hmis-warehouse
+docker-compose run --rm shell bin/setup
+docker-compose run --rm shell bin/db_prep
 
-services:
-  db:
-   ports:
-     - 5432:5432
-   command: ['postgres', '-c', 'max_connections=500', '-c', 'log_statement=all', '-c', 'port=5432']
+cd ../boston-cas
+docker-compose run --rm shell bin/setup
 ```
 
-Then add this to `docker-compose.override.yml` to CAS.
-```yaml
-version: '3.8'
+## Extra Steps for Parallels
 
-services:
-  db:
-    ports:
-      - 5433:5433
-    command: ['postgres', '-c', 'max_connections=500', '-c', 'log_statement=all', '-c', 'port=5433']
-  redis:
-    ports:
-      - 6378 # hide redis from the warehouse
+You may need to run this from inside the Vagrant box to fix DNS issues:
+
+```sh
+# Add google nameservers
+sudo su
+echo 'prepend domain-name-servers 8.8.8.8, 8.8.4.4;'  >> /etc/dhcp/dhclient.conf
+# Run dhclient for the changes to take effect
+dhclient
+# Make sure you can ping google from the vagrant box
+ping google.com
+
+# When your docker containers are up, make sure you can ping google from inside them
+cd boston-cas
+docker-compose run --rm shell ping google.com
+```
```

Comment:
So @eanders, the fix we tried earlier (`systemd-resolve  --set-dns=8.8.8.8 --interface=eth0`) actually doesn't work long-term, because the resolv conf gets overwritten by DHCPd.

I followed [these steps](https://kifarunix.com/make-permanent-dns-changes-on-resolv-conf-in-linux/#dns-in-dhclient.conf) to arrive at the above solution.. I don't know if it's the best way, but it is working for me. I put it here in the docs instead of in the bootstrap script because I'm not sure if it is necessary for virtualbox setups.